compiler-rt/lib/scudo/standalone/primary32.h

```diff
@@ -387,7 +387,7 @@ template <typename Config> class SizeClassAllocator32 {
 
   struct ReleaseToOsInfo {
     uptr BytesInFreeListAtLastCheckpoint;
-    uptr RangesReleased;
+    uptr NumReleaseAttempted;
```

Comment:
This is such a nit, but could this be NumReleasesAttempted?

Reply:
Done